### ISSUE TITLE
Compact mode doesn't show the new pouch.

### ISSUE THREAD

**Issue by maviscass** [Xeio/WanderLost#193]
Heya! The compact mode doesn't show the new pouch.
Not really a big deal but I just noticed it. 

Thanks for your hard work! 
![image](https://github.com/Xeio/WanderLost/assets/104755792/c8ecaa82-02d7-42e6-9955-70acbfecbef1)
![image](https://github.com/Xeio/WanderLost/assets/104755792/b11af004-1682-41c5-853c-8f952ed209cf)

**Comment by Xeio** [comment]
Yeah, it's sort of a todo at the moment, it will at least play the notification sound (if you set those up) for the pouches even on compact mode, but I'm not sure exactly where to fit displaying it. 🤔
